collection-scripts/gather_profiling_node

```diff
@@ -3,10 +3,13 @@
 BASE_COLLECTION_PATH="${BASE_COLLECTION_PATH:-must-gather}"
 PROFILING_NODE_LOG_PATH="${BASE_COLLECTION_PATH}/pprof"
 PROFILING_NODE_KUBELET_LOG_PATH="${PROFILING_NODE_LOG_PATH}/kubelet"
-
+PROFILING_NODE_CRIO_LOG_PATH="${PROFILING_NODE_LOG_PATH}/crio"
 
 PROFILING_NODE_APISERVER="kubernetes.default.svc"
 PROFILING_NODE_SECONDS="${PROFILING_NODE_SECONDS:-30}"
+PROFILING_NODE_TARGET="${PROFILING_NODE_TARGET:-ALL}"
+
+IMAGE_STREAM="${IMAGE_STREAM:-"registry.redhat.io/rhel8/support-tools"}"
```

Comment:
This isn't an Image-Stream but an image; lets rename this so that it reflects what it actually is.

Reply:
I ended up with "PROFILING_NODE_IMAGE"  :-P